homeassistant/components/sensor/dht.py

```diff
@@ -45,6 +46,10 @@
     vol.Optional(CONF_MONITORED_CONDITIONS, default=[]):
         vol.All(cv.ensure_list, [vol.In(SENSOR_TYPES)]),
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_TEMPERATURE_OFFSET, default=0):
+        vol.All(vol.Coerce(float), vol.Range(min=-500, max=1000)),
```

Comment:
Remove the range. It doesn't make much sense to allow -500 °C while the absolute zero point is -273.15 °C. As far as I remember is the measurement range of a DHT11 something like 0-50 ℃